lib/repl.js

```diff
@@ -1222,7 +1222,7 @@ REPLServer.prototype.convertToContext = function(cmd) {
   }
 
   return cmd;
-};
+}, 'replServer.convertToContext will be removed in v8.0.0');
```

Comment:
use "after", that represents a range, instead of the precise word "in" would be better?